Apps/WireGuardEasy/docker-compose.yml

```
@@ -1,7 +1,7 @@
 name: wg-easy
 services:
   wg-easy:
-    image: weejewel/wg-easy:7
+    image: ghcr.io/wg-easy/wg-easy
```

```suggestion
    image: ghcr.io/wg-easy/wg-easy:13
```
The latest version is 13: https://github.com/wg-easy/wg-easy/pkgs/container/wg-easy/217936898?tag=13

---

@CorrectRoadH I've added version, please take a look ^^

---

LGTM. Very Thanks for your contribution.